support/COMMUNITY_CONTACTS.md

```diff
@@ -51,41 +51,56 @@ only expected to participate during their local office hour.
 
 # Roster
 
-We seed this rotation with all approvers from all the Eventing workgroups,
-excluding productivity. If you are no longer active in Knative, or if you are
-contributing on personal capacity and do not have time to contribute in the
-rotation, feel free to send a PR to remove yourself. Similarly you can trade
-your timeslot with others by sending PRs.
+We seed this rotation with all approvers and reviewers from all the
+Eventing workgroups, excluding productivity. If you are no longer
+active in Knative, or if you are contributing on personal capacity and
+do not have time to contribute in the rotation, feel free to send a PR
+to remove yourself and add yourself to the NON_ROSTER file. Similarly
+you can trade your timeslot with others by sending PRs.
+
+The schedule can be regenerated randomly for a new cycle with the
+command (run from the eventing base directory)
+
+```sh
+grep "^- " OWNERS | cut -f2 -d" " | grep -v -f support/NON_ROSTER | sort | uniq | sort -R
+```
 
 - [grantr](https://github.com/grantr)
 - [Harwayne](https://github.com/Harwayne)
 - [lionelvillard](https://github.com/lionelvillard)
 - [matzew](https://github.com/matzew)
 - [n3wscott](https://github.com/n3wscott)
-- [nachocano](https://github.com/nachocano)
 - [slinkydeveloper](https://github.com/slinkydeveloper)
 - [vaikas](https://github.com/vaikas)
 - [lberk](https://github.com/lberk)
 - [pierDipi](https://github.com/pierDipi)
 - [aliok](https://github.com/aliok)
+- [aslom](https://github.com/aslom)
+- [tayarani](https://github.com/tayarani)
+- [zhongduo](https://github.com/zhongduo)
+- [liu-cong](https://github.com/liu-cong)
```

Comment:
liu-cong is not working on this project any more, we will remove from the reviewer list. Can you please rerun the tool later? Thanks.

Reply:
Done, and added to the non_roster file for now.

Will rebase on the updated schedule once that merges